extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -96,10 +245,63 @@ class AmpDailymotion extends AMP.BaseElement {
 
   /** @override */
   pauseCallback() {
-    if (this.iframe_ && this.iframe_.contentWindow) {
-      this.iframe_.contentWindow./*OK*/postMessage('pause', '*');
+    if (this.iframe_ && this.iframe_.contentWindow
```

Comment:
For some reason git doesn't think this was fixed, but it was fixed in the last commit.